src/commands/pod_install.yml

```diff
@@ -21,7 +21,7 @@ steps:
         - restore_cache:
             keys:
               - cache-pods-{{ checksum "<<parameters.pod_install_directory>>/Podfile.lock" }}-{{ .Environment.CACHE_VERSION }}
-              - cache-pods-
+              - cache-pods-{{ .Environment.CACHE_VERSION }}
```

Comment:
```suggestion
              - cache-pods-{{ .Environment.CACHE_VERSION }}-{{ checksum "<<parameters.pod_install_directory>>/Podfile.lock" }}
              - cache-pods-{{ .Environment.CACHE_VERSION }}-
              - cache-pods-
```

Reply:
@cortinico yes, indeed! 🤦 

I need to move the `CACHE_VERSION` at the front otherwise it will never match. Thought about it a week ago when I was looking into the problem and totally forgot about it today. 😅

By the way, I think having the `CACHE_VERSION` as part of the prefix is a good practice for all cache keys. I'll just change this one now and I can open another PR for the broader change, if you agree.

Reply:
Done. ✅

Reply:
Hmmm… the proposal is not properly bust the cache.

If we increment the `CACHE_VERSION` then the first 2 keys will miss but the last one will match. In that case the "old" pods will be fetched and `pod install` will fail again (for `RTC-Folly`), right?

So, I removed the 3rd cache key. What do you think?

Reply:
Yup you're right. We should however use something like:

```
- v{{ .Environment.CACHE_VERSION }}-cache-pods-
```

To follow the same approach used in the CircleCI docs and in the react-native repo https://circleci.com/docs/2.0/caching/#restoring-cache

Reply:
> By the way `react-native` is [not consistent](https://github.com/facebook/react-native/blob/v0.67.3/.circleci/config.yml?rgh-link-date=2022-02-25T18%3A07%3A55Z#L71-L74) with the `v` prefix [either](https://github.com/facebook/react-native/blob/v0.67.3/.circleci/config.yml?rgh-link-date=2022-02-25T18%3A07%3A55Z#L81-L94). 😅

Yup you're right :) LGTM

Reply:
Resolving  this thread. Would love to see a new published version.  🙏